src/Components/RatingList/RatingList.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  margin-bottom: 20px;
+  button {
+    border: 0;
+    outline: 0;
+    background: transparent;
+    text-align: center;
+    font-family: inherit;
+    padding: 0;
+  }
+  margin-top: 8px;
+  .emojiRating {
+    padding: 3px;
+    margin-right: 8px;
+    font-size: 16px;
+    background: #cccccc;
+  }
+  .numberRating {
+    margin-top: 4px;
+    margin-right: 8px;
+  }
+`;
+
+const RatingList = () => (
+  <Wrapper>
+    <button>
+      <div className="emojiRating">🙁</div>
```

Comment:
I think it's better to use images instead of the actual emojis here because they look diff not on Mac - I can give u the images if u want :)

Reply:
Sounds good 👍👍

Reply:
@leeandher they're available at src/Assets if u pull from master

Reply:
I updated the emojis to be rapped in a <span>